### ISSUE TITLE
Create Tournament Percentage Bar For Completion

### ISSUE THREAD

**Issue by NikAlexander** [BattlePro/battlepro-issues#265]
When organizers are creating tournaments, I would like the level of completion to be really easy to understand.  Please create a percentage bar that shows the completion amount of a newly created tournament or event.

Concept:
![tournament progress](https://user-images.githubusercontent.com/6005869/36939311-b2a1a218-1efc-11e8-8898-9296b2861fff.jpg)

**Comment by mikecreuzer** [comment]
Yeah it would depend on what all we needed them to fill out. Right now we ask for required information on create, anything after that is not required. So theoretically a progress bar isn't exactly needed with the scope of what exists.

That said, an ideal progress bar to me would be something like:

1. Create tournament (setup required fields)
2. Setup teams (or individuals)
3. Each member confirms times, terms, and payouts

Or something to that effect. @NikAlexander

**Comment by NikAlexander** [comment]
If its a bit tricky to implement perhaps just splitting the percentage bar into four 25% segments would work.  User would gain 25% after each stage is complete:
- basics
- game info
- tournament details
- staging
